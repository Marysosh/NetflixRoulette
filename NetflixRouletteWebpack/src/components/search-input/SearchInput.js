import React from "react";
import "./SearchInput.scss";

class SearchInput extends React.PureComponent {
  render() {
    return (
      <input
        className="search-input"
        type="text"
        placeholder="What do you want to watch?"
      />
    );
  }
}
export default SearchInput;
