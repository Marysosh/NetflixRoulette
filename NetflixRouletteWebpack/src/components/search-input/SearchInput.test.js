import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";

import SearchInput from "./SearchInput";

describe("Search input", () => {
  it("should render correctly", () => {
    const { asFragment } = render(
      <Router>
        <SearchInput />
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
