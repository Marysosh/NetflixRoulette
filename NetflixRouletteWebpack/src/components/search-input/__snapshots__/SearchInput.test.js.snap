// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search input should render correctly 1`] = `
<DocumentFragment>
  <input
    class="search-input"
    placeholder="What do you want to watch?"
    type="text"
    value=""
  />
</DocumentFragment>
`;
